utils/build-script

```diff
@@ -1,15 +1,20 @@
 #!/usr/bin/env python
-# utils/build-script - The ultimate tool for building Swift -*- python -*-
-#
+
 # This source file is part of the Swift.org open source project
 #
-# Copyright (c) 2014 - 2017 Apple Inc. and the Swift project authors
+# Copyright (c) 2014 - 2020 Apple Inc. and the Swift project authors
 # Licensed under Apache License v2.0 with Runtime Library Exception
 #
 # See https://swift.org/LICENSE.txt for license information
 # See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
 
-from __future__ import print_function
+
+"""
+The ultimate tool for building Swift.
```

Comment:
lol, I respectfully disagree, the ultimate tool is CMake!

Reply:
I was just transferring the comment from the top of the file.